nostr.ts

```diff
@@ -247,10 +284,22 @@ export class InMemoryAccountContext implements NostrAccountContext {
         event: UnsignedNostrEvent<Kind>,
     ): Promise<NostrEvent<Kind>> {
         const id = await calculateId(event);
-        const sig = utf8Decode(hex.encode(await signId(id, this.privateKey.hex)));
+        const sig = encodeHex(await signId(id, this.privateKey.hex));
```

Comment:
so the new std has a faster implementation?

Reply:
```ts
const from = new Date()
const sig = encodeHex(await signId(id, this.privateKey.hex));
const to = new Date()
console.log("signEvent dur:", to.getMilliseconds() - from.getMilliseconds(), sig)

const from_old = new Date()
const sig_old = utf8Decode(hex.encode(await signId(id, this.privateKey.hex)));
const to_old = new Date()
console.log("signEvent dur(old):", to_old.getMilliseconds() - from.getMilliseconds(), sig_old)
```

result:
```
signEvent dur: 4 44de8bc13c2544100595310b897e199caad199d8c10919eeb0e15b114309799d546da8d0b489501ac60e1528f91e923bc980c75629116f9ad25a77e8d6365ff2
signEvent dur(old): 8 e42368915c8b4261ff941bed9082d8ad2073b74f80e1aa3c50604e1557bf15f62b3111ec0fe688d4c042a6844d04bf63ada91f22fe176a4f417157446e00f734
signEvent dur: 3 549aa5ca8ac7f3e0df52c54abff165a56775e4191b28f14364e77ce8f80268e45aeb0df71aff3e0d36df4590a9dbf3f9dc5b606e1cb59a0988a42ab495eadfb6
signEvent dur(old): 7 1662ebbebd033b76468b048c582cc3bd4c0b368c5689ee1a23b1c8b0eb389ce1c984ade187c67f8faa82c71013452aa062b345f6dcdff6885d29edd6cf238b8b
```

The new std offers a faster implementation, although the test was not very rigorous.

Reply:
Very good, faster and simpler